calico/felix/config.py

```diff
@@ -406,6 +417,10 @@ def _validate_cfg(self, final=True):
             log.warning("Reporting TTL set to %s.", self.REPORTING_TTL_SECS)
             self.REPORTING_TTL_SECS = self.REPORTING_INTERVAL_SECS * 5/2
 
+        if self.ENDPOINT_REPORT_DELAY < 0:
+            log.warning("Endpoint status delay is negative, defaulting to 1.")
```

Comment:
I think 0 is valid, it means "don't delay, report immediately".  The loop will still block waiting for work.